src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/FullMeansTestOutcomeCalculator.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.FullAssessmentResult;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MagCourtOutcome;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.CaseType;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MeansTestOutcome;
+import java.util.HashSet;
+import java.util.Set;
+
+@Slf4j
+public class FullMeansTestOutcomeCalculator {
+
+    public static MeansTestOutcome getFullMeansTestOutcome(FullAssessmentResult result, CaseType caseType, MagCourtOutcome magCourtOutcome){
+        log.debug("Get the outcome of the Full Means Test result = {} caseType = {} magCourtOutcome = {}", result, caseType, magCourtOutcome);
+
+        //Fail data
+        Set<CaseType> caseTypesHeardInMagistratesCourt = new HashSet<CaseType>();
+        caseTypesHeardInMagistratesCourt .add(CaseType.COMMITAL);
+        caseTypesHeardInMagistratesCourt .add(CaseType.SUMMARY_ONLY);
+        caseTypesHeardInMagistratesCourt .add(CaseType.EITHER_WAY);
+
```

Comment:
we should do this with Set.of so that it can be static rather than being created at function scope each time